eng/dependabot/net8.0/Directory.Build.props

```diff
@@ -1,4 +1,8 @@
 <Project>
   <!-- Dependendabot is configured to read package versions from Directory.Build.props, so just use this file as a shim. -->
   <Import Project="Versions.props" />
+  <Import Project="Packages.props" />
+  <PropertyGroup>
+    <_SuppressSdkImports>true</_SuppressSdkImports>
```

Comment:
The other versioned Directory.Build.props files do not have this setting. Is the needed for this version for some particular reason?

Reply:
This was a bad copy-paste, good catch. Fixed net8.0